pkg/aggregator/mocksender/asserts.go

```diff
@@ -46,6 +47,54 @@ func (m *MockSender) AssertMetricNotTaggedWith(t *testing.T, method string, metr
 	return m.Mock.AssertCalled(t, method, metric, mock.AnythingOfType("float64"), mock.AnythingOfType("string"), AssertTagsNotContains(tags))
 }
 
+// Compare an Event on specifics values
+func eventHaveEqualValues(expectedEvent, actualEvent metrics.Event) bool {
```

Comment:
I'd use a `...Like` / `...Alike` name as we don't test strict equality. Please document the things we don't test